public/locales/en/translation.json

```diff
@@ -219,16 +222,30 @@
     },
     "projects": {
         "projects": "project",
+        "cvprojects": "project",
         "addProject": "Add project",
         "updateProject": "Update  project",
         "deleteProject": "Delete project",
         "name": "Name",
-        "internalName": "Internal Name",
+        "internal_name": "Internal Name",
         "domain": "Domain",
         "teamSize": "Team size",
         "description": "Description",
-        "startDate": "Start date",
-        "endDate": "End date",
+        "start_date": "Start date",
+        "end_date": "End date",
```

Comment:
why to use underscore in translation files if we all user camelCase from the very beginning?

Reply:
that naming used in library, and i have to use same naming to parse data for the table